response.md

```diff
@@ -1,12 +1,12 @@
 # Search results
 
-When requesting results from Mapzen Search, you will always get back `GeoJSON` results, unless something goes terribly wrong, in which case you'll get an error message.
+When requesting results from Mapzen Search, you will always get back GeoJSON results, unless something goes terribly wrong, in which case you'll get an error message.
 
   Tip: You can go to http://geojson.org/geojson-spec.html to learn more about the GeoJSON data format specification.
 
 The top-level structure to every response looks like this:
 
-```json
```

Comment:
I removed it for now because it was causing formatting problems with the markdown parser, where the [...] was being shown in a red block and it was hard to read. I'll have to investigate json with our parser.